sdk/lib/cmlib/hivewrt.c

```diff
@@ -182,17 +186,13 @@ HvpWriteLog(
         /* Check if the block is clean or we're past the last block */
         LastIndex = BlockIndex;
         BlockIndex = RtlFindSetBits(&RegistryHive->DirtyVector, 1, BlockIndex);
-        if (BlockIndex == ~HV_CLEAN_BLOCK || BlockIndex < LastIndex)
+        if (BlockIndex == 0xFFFFFFFF || BlockIndex < LastIndex)
         {
             break;
         }
 
         /*
          * Mark this block as dirty and go to the next one.
-         *
-         * FIXME: We should rather use RtlSetBits but that crashes
```

Comment:
Yeah probably. I tend to think making this an RTL_BITMAP is overkill when we're using a "byte-map" anyway. But @GeoB99 indicated he might still want to do this, so I can leave the comment here.

Reply:
we may be currently using a byte-map but this is not going to generate windows-compatible LOG files, and it would be preferrable to change that to a more windows-compatible implementation of that in the future. So keeping the comment would be better I think, in order to indicate this fact.

Reply:
AFAIU, this is exactly Windows-compatible. It's the one-bit-per-page mechanism that's not compatible, which is why I went in this direction with the fix (as opposed to changing the assignment to only set one bit).

Reply:
It's explained there: https://github.com/msuhanov/regf/blob/master/Windows%20registry%20file%20format%20specification.md#format-of-transaction-log-files

Reply:
Right now we're not compatible with anything -- we write 0xFF per 4096 byte block, but we make the bitmap too small to fit all the blocks. So if we're starting out being compatible, presumably it should be Win2003?